CONTRIBUTORS

```diff
@@ -67,3 +67,4 @@ monjiwired@
 Philippe Liard
 Sang-il Lee
 Yugui Sonoda
+Amir Farsi
```

Comment:
The list is sorted alphabetically. Please add your name after Alexandru / before András.

Thanks.